cluster_tools/dockered-slurm/cgroup.conf

```diff
@@ -0,0 +1,3 @@
+# autodetect which is the default detects cgroup/v2 in the github CI,
+# which fails during the initialization of the c1 and c2 nodes
+CgroupPlugin=cgroup/v1
```

Comment:
kudos! how did you find this? was there a hint about this in the logs?

Reply:
The initialization of the c1 and c2 nodes failed but there wasn't any hint on what went wrong. Therefore, I added

```
# Show log output for debugging
docker logs slurmctld
docker logs c1
docker logs c2
```

to the ci.yml which outputs the initialization log. In there I saw

```
slurmd: error: Couldn't find the specified plugin name for cgroup/v2 looking at all files
slurmd: error: cannot find cgroup plugin for cgroup/v2
slurmd: error: cannot create cgroup context for cgroup/v2
slurmd: error: Unable to initialize cgroup plugin
slurmd: error: slurmd initialization failed
```

and a quick local cross-check showed that at least on my laptop cgroup/v1 is used instead, successfully :)

I left the `docker logs ...` statements in there as they don't do any harm and might help again in the future.

Reply:
great find :) and :+1: for the added logs.